mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go

```diff
@@ -3186,6 +3214,34 @@ data "google_sql_backup_run" "backup" {
 `, context)
 }
 
+func testAccSqlDatabaseInstance_cloneWithDatabaseNames(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_sql_database_instance" "instance" {
+  name             = "tf-test-%{random_suffix}"
+  database_version = "POSTGRES_11"
```

Comment:
The documentation states database_names can only be used with SQL Server. How does this work with Postgres?

Reply:
Postgres will ignore this attribute. The test util `BootstrapSharedSQLInstanceBackupRun()` sets up with POSTGRES_11 so I have to use the same database type here as well. The test sends out the field from the client, but the test does not verify data in the clone result, so it does not make any difference with which database type to use.